frontend-student/src/components/coursePage/AISummaryComponent.jsx

```
@@ -0,0 +1,238 @@
+        }
+      );
+
+      const data = await response.json();
+
+      if (!response.ok || !data.candidates || data.candidates.length === 0) {
+        throw new Error(data.error?.message || 'Failed to generate summary');
+      }
+
+      const summaryText =
+        data.candidates[0].content?.parts[0]?.text || 'No summary generated';
+      const formattedSummary = 'AI Summary\n\n' + summaryText;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)